xml/gl.xml

```diff
@@ -5234,7 +5234,7 @@ typedef unsigned int GLhandleARB;
         <enum value="0" name="GL_FALSE" group="Boolean,VertexShaderWriteMaskEXT,ClampColorModeARB"/>
         <enum value="0" name="GL_NO_ERROR" group="GraphicsResetStatus,ErrorCode"/>
         <enum value="0" name="GL_ZERO" group="TextureSwizzle,StencilOp,BlendingFactor"/>
-        <enum value="0" name="GL_NONE" group="TextureCompareMode,PathColorFormat,CombinerBiasNV,CombinerScaleNV,DrawBufferMode,PixelTexGenMode,ReadBufferMode,ColorBuffer,PathGenMode,PathTransformType,PathFontStyle"/>
+        <enum value="0" name="GL_NONE" group="EmptyFlags,TextureCompareMode,PathColorFormat,CombinerBiasNV,CombinerScaleNV,DrawBufferMode,PixelTexGenMode,ReadBufferMode,ColorBuffer,PathGenMode,PathTransformType,PathFontStyle"/>
```

Comment:
I wouldn't call this `EmptyFlags` as it's not future-proof. Perhaps `SyncBehaviorFlags`? (Yes, I know that there's no currently defined flags for the glFenceSync flags parameter, but maybe one day there will be...)

Reply:
Well, this only part of core functions, i imagine there would be more of empty enum/bitfield arguments in extensions.

They probably not getting any enum values any time soon, so it makes sense to use 2 common groups in all functions - `EmptyEnum` and `EmptyFlags`, to not clog up xml files. And then change group name if these functions do get enum values.

It shouldn't be any problem to change group name later, right?

Reply:
I really think we should keep GLenum and GLbitfield parameters without associated enumerants untagged, Empty* is a smell and a little pointless. Group changes later down the line are technically breaking if concerning the function signatures.

Reply:
> Group changes later down the line are technically breaking if concerning the function signatures.

Well, adding missing `group="GetPName"` would also break signatures then...
But i guess changes that break signatures can at least be minimized.